### ISSUE TITLE
What library is used to develop GUI application and game?

### ISSUE THREAD

**Issue by feiyuhuahuo** [honghaier-game/PyMe#57]
GUI application development is based on Tkinter and game is based on PyGame, did I get it right? Will PyMe support PyGame-ce?

**Comment by honghaier-game** [comment]
YES,GUI is based om tkinter.but i consider to use other lib, example :ttkbootstrap,or customtkinter. 
2D Game is based on pygame ,3D Game will based on PyOpenGL, but it hasn't been developed yet.
PyGame-ce ? I dont't know it. thanks for reminder.
